src/test/java/com/google/api/codegen/testdata/discoveries/py/py_storage.v1.json.baseline

```diff
@@ -1320,31 +1137,22 @@ from pprint import pprint
 from googleapiclient import discovery
 from oauth2client.client import GoogleCredentials
 
-
-# Use Application Default Credentials for authentication when running locally.
-# For more information, see:
-# https://developers.google.com/identity/protocols/application-default-credentials
 credentials = GoogleCredentials.get_application_default()
 
 service = discovery.build('storage', 'v1', credentials=credentials)
 
-
-# TODO: Change placeholder below to desired parameter value for the `list` method:
-
 # Name of the bucket in which to look for objects.
-bucket = '{MY-BUCKET}'
-
+bucket = ''  # TODO: Update placeholder value.
 
-objects = service.objects()
-request = objects.list(bucket=bucket)
+request = service.objects().list(bucket=bucket)
 while request is not None:
     response = request.execute()
 
-    for object_ in response['items']:
-        # TODO: Change code below to process each `object_` resource:
-        pprint(object_)
+    for object in response['items']:
```

Comment:
If we use built-ins as local variables, will they be highlighted undesirably in API Explorer?